subtests/docker_cli/systemd/systemd_pull.py

```diff
@@ -0,0 +1,28 @@
+"""
+Pull docker images using systemd unit file - docker-pull.service is used.
+This unit file is copied to system and minor edition is made. A systemd
+service is started to pull specified image.
+"""
+
+from systemd import systemd_base
+from dockertest.images import DockerImages
+# from dockertest.config import ConfigSection
+
+
+class systemd_pull(systemd_base):
+
+    def initialize(self):
+        super(systemd_pull, self).initialize()
+        # DockerImages(self).clean_all([self.config['image_to_pull']])
+        DockerImages(self).clean_all([self.config['image_name']])
```

Comment:
Some reason ``clean_all()`` needs to happen twice (once in ancestor's ``initialize()`` and another here)?